lib/avromatic/schema_registry_patch.rb

```diff
@@ -0,0 +1,23 @@
+require 'avro_turf'
+require 'avro_turf/schema_registry'
+
+AvroTurf::SchemaRegistry.class_eval do
+  # Override register to first check if a schema is registered by fingerprint
+  def register(subject, schema)
+    raise 'schema must be an Avro::Schema' unless schema.is_a?(Avro::Schema)
```

Comment:
I was thinking that an `Avro::Schema` object was required because we need to get a fingerprint from it for the first request, we always pass a schema object using this library. But if passed a string I guess we could attempt to parse it here.